packages/cli/__tests__/report-components/table-test.ts

```diff
@@ -45,11 +51,67 @@ describe('the correct data is rendered for a table', () => {
                     <td class=\\"border px-4 py-2\\">12.2.2</td>
                 </tr>
                 <tr class=\\"bg-gray-100\\">
-                    <td class=\\"border px-4 py-2\\">ember-data</td>
+                    <td class=\\"border px-4 py-2\\">yarn</td>
                     <td class=\\"border px-4 py-2\\">4.2.2</td>
                 </tr>
                 <tr class=\\"bg-gray-100\\">
-                    <td class=\\"border px-4 py-2\\">ember-qunit</td>
+                    <td class=\\"border px-4 py-2\\">qunit</td>
+                    <td class=\\"border px-4 py-2\\">6.1.2</td>
+                </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      "
+    `);
+  });
+});
+
+describe('the correct data is rendered for a graded table', () => {
+  const expectedOutput: { meta: TaskMetaData; result: DependencyResult[] } = {
+    meta: {
+      taskName: 'mock-task',
+      friendlyTaskName: 'Mock Task',
+      taskClassification: {
+        category: Category.Migrations,
+        priority: Priority.Medium,
+      },
+    },
+    result: [
+      { name: 'node', value: '12.2.2', grade: Grade.A },
+      { name: 'yarn', value: '4.2.2', grade: Grade.B },
```

Comment:
```suggestion
      { name: 'foo', value: '12.2.2', grade: Grade.A },
      { name: 'bar', value: '4.2.2', grade: Grade.B },
```

I don't think we'd add `node` or `yarn` as dependencies...